soperator/installations/example/main.tf

```diff
@@ -68,6 +68,24 @@ module "filestore" {
   }
 }
 
+module "nfs-server" {
+  count          = var.nfs.enabled ? 1 : 0
+  source         = "../../../modules/nfs-server"
+  parent_id      = data.nebius_iam_v1_project.this.id
+  subnet_id      = data.nebius_vpc_v1_subnet.this.id
+  ssh_user_name  = "soperator"
+  ssh_public_key = var.slurm_login_ssh_root_public_keys[0]
+  nfs_ip_range   = data.nebius_vpc_v1_subnet.this.ipv4_private_pools.pools[0].cidrs[0].cidr
+  nfs_size       = var.nfs.size_gibibytes * 1024 * 1024 * 1024
+  nfs_path       = "/mnt/nfs"
+  platform       = "cpu-e2"
+  preset         = "16vcpu-64gb"
```

Comment:
We should also make the preset configurable.

Reply:
Done